lib/rdoc/markup.rb

```diff
@@ -10,19 +10,24 @@
 # RDoc::Markup and other markup formats do no output formatting, this is
 # handled by the RDoc::Markup::Formatter subclasses.
 #
-# = Supported Formats
-#
-# Besides the RDoc::Markup format, the following formats are built in to RDoc:
-#
-# markdown::
-#   The markdown format as described by
-#   http://daringfireball.net/projects/markdown/.  See RDoc::Markdown for
-#   details on the parser and supported extensions.
-# rd::
-#   The rdtool format.  See RDoc::RD for details on the parser and format.
-# tomdoc::
-#   The TomDoc format as described by http://tomdoc.org/.  See RDoc::TomDoc
-#   for details on the parser and supported extensions.
+# = Markup Formats
+#
+# +RDoc+ supports these markup formats:
+#
+# - +rdoc+:
+#   the +RDoc+ markup format;
+#   see RDoc::Markup.
```

Comment:
Doesn't this just link to the current file? There's not much information on that file. Isn't it more useful to link to `MarkupReference`?

Reply:
I'm confused. This bullet point is for RDoc markup, not markdown.

Reply:
Duh.  I'm the one who was confused.  Now fixed to refer to RDoc::MarkupReference.